### PR TITLE
chore: v7.0.0 release

### PR DESCRIPTION
**Pull request by nfriedly**
I took the `next` branch, rebased against `main`, fixed several conflicts, and updated the changelog. I decided to do a PR so that we could both give it an extra pass before merging into `main` and releasing.

This PR should be merged rather than "squash and merge"d

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -547,10 +565,12 @@ if you wish to create your own store.
 ## Request API
 
 A `request.rateLimit` property is added to all requests with the `limit`,
```
Comment (nfriedly) [inline readme.md:567]: I'm fine with full names in our codebase, but perhaps we should rename `request` to `req` in our docs to match Express?

> In this documentation and by convention, the object is always referred to as req (and the HTTP response is res) 

http://expressjs.com/en/4x/api.html#req

Comment (gamemaker1) [reply readme.md:567]: Ohkay, sure!

**Review by gamemaker1 — APPROVED** [review]
LGTM!

Let's get this released soon! :shipit:

**Comment by gamemaker1** [comment]
Let me know if I should merge the PR

**Comment by nfriedly** [comment]
Yeah, I think this is ready to go. Would you like to do the honors?

**Comment by gamemaker1** [comment]
It would be my pleasure :D

**Pull request merged**